barretenberg/cpp/src/barretenberg/vm/avm/generated/flavor.hpp

```diff
@@ -135,6 +135,10 @@ class AvmFlavor {
     // We have two copies of the witness entities, so we subtract the number of fixed ones (they have no shift), one for
     // the unshifted and one for the shifted
     static constexpr size_t NUM_ALL_ENTITIES = 787;
+    static constexpr size_t NUM_ALL_WITNESS_ENTITIES = 771;
```

Comment:
Apologies that this got removed. It is in our codegen and will be regenerated next time.